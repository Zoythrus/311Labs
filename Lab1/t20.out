0
4
8
12
18
19
20
24
27
35
38
40
42
42
43
44
44
45
46
56
61
63
64
65
68
70
71
77
77
78
82
85
87
90
91
93
94
95
