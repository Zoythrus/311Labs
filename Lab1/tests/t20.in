stat
20 20 100 difference
4
20
27
38
43
42
44
45
56
61
63
70
77
78
90
87
91
93
96
100
8
0
42
46
18
19
82
85
40
71
64
35
68
12
77
94
65
44
95
24
