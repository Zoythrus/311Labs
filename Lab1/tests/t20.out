4
20
27
38
43
45
56
61
63
70
78
87
90
91
93
96
